(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[867],{1558:function(e,i,n){(window.__NEXT_P=window.__NEXT_P||[]).push(["/research2",function(){return n(4455)}])},4455:function(e,i,n){"use strict";n.r(i);var s=n(5893);n(7294),i.default=()=>(0,s.jsx)("div",{id:"page-wrapper",children:(0,s.jsx)("div",{className:"wrapper style1",children:(0,s.jsxs)("section",{id:"main",className:"container",children:[(0,s.jsx)("header",{className:"major",children:(0,s.jsx)("h3",{children:"Ongoing research"})}),(0,s.jsxs)("div",{className:"row",children:[(0,s.jsx)("div",{className:"8u 12u(narrower)",children:(0,s.jsxs)("section",{id:"content",className:"box",children:[(0,s.jsx)("a",{href:"#",className:"image fit",children:(0,s.jsx)("img",{src:"images/research2.png",alt:""})}),(0,s.jsx)("h4",{children:"Artificial Intelligence and deep learning models for biochemical applications"}),(0,s.jsx)("p",{children:"The Kannan lab is exploring cutting-edge AI research in the field of protein function prediction. One key area is understanding and predicting post-translational modifications, which are critical to regulating cellular processes. We developed a unified framework (Phosphormer) [1] using protein language models to generate context-aware features from primary protein sequence data to predict kinase-specific phosphosites. We are working on enabling this model to make accurate phosphosite predictions across the entire kinome and expand to other post-translational modifications. We have further developed workflows and visualization methods to classify protein families using sequence embeddings or feature-rich numerical representations of protein sequences. These embeddings infer proteins' structural and functional properties based solely on sequence information. We successfully applied this in the classification of phosphatases, protein kinases, and the S-Adenosyl-L-Methionine (SAM) enzyme superfamily, generating Neighbor-Joining (NJ) embedding trees as an orthogonal method to phylogeny for visualization of evolutionary information [2, 3, 4]. These studies highlight AI's transformative power in protein science, offering significant insights into protein structure, function, and dynamics."})]})}),(0,s.jsx)("div",{className:"4u 12u(narrower)",children:(0,s.jsx)("section",{id:"sidebar",className:"box",children:(0,s.jsxs)("section",{children:[(0,s.jsx)("h3",{children:"Selected Publications:"}),(0,s.jsxs)("ol",{children:[(0,s.jsxs)("li",{children:[(0,s.jsx)("a",{href:"https://academic.oup.com/bioinformatics/article/39/2/btad046/7000331",children:"Phosformer: an explainable transformer model for protein kinase-specific phosphorylation predictions"}),(0,s.jsx)("hr",{})]}),(0,s.jsxs)("li",{children:[(0,s.jsx)("a",{href:"https://academic.oup.com/bib/article/24/1/bbac619/6987820",children:"Tree visualizations of protein sequence embedding space enable improved functional clustering of diverse protein superfamilies"}),(0,s.jsx)("hr",{})]}),(0,s.jsxs)("li",{children:[(0,s.jsx)("a",{href:"https://academic.oup.com/bib/article/24/1/bbac599/6984799",children:"Alignment-free estimation of sequence conservation for identifying functional sites using protein sequence embeddings"}),(0,s.jsx)("hr",{})]}),(0,s.jsxs)("li",{children:[(0,s.jsx)("a",{href:"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8476585/",children:"Mapping the glycosyltransferase fold landscape using interpretable deep learning"}),(0,s.jsx)("br",{}),(0,s.jsx)("br",{}),(0,s.jsx)("a",{href:"publications",className:"button alt",children:"View all publications"}),(0,s.jsx)("hr",{})]})]})]})})})]})]})})})}},function(e){e.O(0,[888,774,179],function(){return e(e.s=1558)}),_N_E=e.O()}]);