(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[149],{1776:function(e,s,i){(window.__NEXT_P=window.__NEXT_P||[]).push(["/research1",function(){return i(6014)}])},6014:function(e,s,i){"use strict";i.r(s);var n=i(5893);i(7294),s.default=()=>(0,n.jsx)("div",{children:(0,n.jsx)("div",{id:"page-wrapper",children:(0,n.jsx)("div",{className:"wrapper style1",children:(0,n.jsxs)("section",{id:"main",className:"container",children:[(0,n.jsx)("header",{className:"major",children:(0,n.jsx)("h3",{children:"Ongoing research"})}),(0,n.jsxs)("div",{className:"row",children:[(0,n.jsx)("div",{className:"8u 12u(narrower)",children:(0,n.jsxs)("section",{id:"content",className:"box",children:[(0,n.jsx)("a",{href:"#",className:"image fit",children:(0,n.jsx)("img",{src:"images/research1.png",alt:""})}),(0,n.jsx)("h4",{children:"Mapping genome-phenome relationships in large protein families"}),(0,n.jsx)("p",{children:"The Kannan lab is focused on pursuing rigorous and deep biological questions connecting protein sequence, structure, function, and regulation as an essential step toward inferring causal genome-phenome relationships. We focus on large gene families such as protein kinases, glycosyltransferases, and ion channels that contribute to diverse disease phenotypes such as cancer, diabetes, Alzheimerâ€™s, and Parkinson's'. We deploy systems approaches combining computational and experimental techniques to generate and test models of protein structure, function, evolution, and dynamics. We aim to develop personalized therapeutic strategies based on a deeper understanding of how these proteins work in disease and normal states. Our studies have provided new mechanistic insights into the unique modes of regulation in various protein kinases [1, 2], small molecule kinases [3], and pseudokinases [4] and have revealed the mechanism of action of oncogenic mutations in receptor tyrosine kinases [5]. We have also successfully employed the specialized tools and approaches developed for studying protein kinases to glycosyltransferases [6, 7] and are currently extending these approaches to studying ion channels."})]})}),(0,n.jsx)("div",{className:"4u 12u(narrower)",children:(0,n.jsx)("section",{id:"sidebar",className:"box",children:(0,n.jsxs)("section",{children:[(0,n.jsx)("h3",{children:"Selected Publications:"}),(0,n.jsxs)("ol",{children:[(0,n.jsxs)("li",{children:[(0,n.jsx)("a",{href:"https://www.biorxiv.org/content/biorxiv/early/2023/03/29/2023.03.29.534689.full.pdf",children:"Mechanistic and evolutionary insights into isoform-specific 'supercharging' in DCLK family kinases"}),(0,n.jsx)("hr",{})]}),(0,n.jsxs)("li",{children:[(0,n.jsx)("a",{href:"https://academic.oup.com/bioinformatics/article/39/2/btad046/7000331",children:"Evolution of Functional Diversity in the Holozoan Tyrosine Kinome"}),(0,n.jsx)("hr",{})]}),(0,n.jsxs)("li",{children:[(0,n.jsx)("a",{href:"https://www.science.org/doi/10.1126/scisignal.aax6313?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed",children:"A redox-active switch in fructosamine-3-kinases expands the regulatory repertoire of the protein kinase superfamily"}),(0,n.jsx)("hr",{})]}),(0,n.jsxs)("li",{children:[(0,n.jsx)("a",{href:"https://www.science.org/doi/10.1126/scisignal.aav3810?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed",children:"Tracing the origin and evolution of pseudokinases across the tree of life"}),(0,n.jsx)("hr",{})]}),(0,n.jsxs)("li",{children:[(0,n.jsx)("a",{href:"https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003545",children:"Prediction and Prioritization of Rare Oncogenic Mutations in the Cancer Kinome Using Novel Features and Multiple Classifiers"}),(0,n.jsx)("hr",{})]}),(0,n.jsxs)("li",{children:[(0,n.jsx)("a",{href:"https://www.jbc.org/article/S0021-9258(22)00654-8/fulltext",children:"Modularity of the hydrophobic core and evolution of functional diversity in fold A glycosyltransferases"}),(0,n.jsx)("hr",{})]}),(0,n.jsxs)("li",{children:[(0,n.jsx)("a",{href:"https://elifesciences.org/articles/54532",children:"Deep evolutionary analysis reveals the design principles of fold A glycosyltransferases"}),(0,n.jsx)("br",{}),(0,n.jsx)("br",{}),(0,n.jsx)("a",{href:"/publications",className:"button alt",children:"View all publications"}),(0,n.jsx)("hr",{})]})]})]})})})]})]})})})})}},function(e){e.O(0,[888,774,179],function(){return e(e.s=1776)}),_N_E=e.O()}]);