(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[84],{2682:function(e,n,i){(window.__NEXT_P=window.__NEXT_P||[]).push(["/research3",function(){return i(2488)}])},2488:function(e,n,i){"use strict";i.r(n);var a=i(5893);i(7294),n.default=()=>(0,a.jsx)("div",{id:"page-wrapper",children:(0,a.jsx)("div",{className:"wrapper style1",children:(0,a.jsxs)("section",{id:"main",className:"container",children:[(0,a.jsx)("header",{className:"major",children:(0,a.jsx)("h3",{children:"Ongoing research"})}),(0,a.jsxs)("div",{className:"row",children:[(0,a.jsx)("div",{className:"8u 12u(narrower)",children:(0,a.jsxs)("section",{id:"content",className:"box",children:[(0,a.jsx)("a",{href:"#",className:"image fit",children:(0,a.jsx)("img",{src:"images/research3.png",alt:""})}),(0,a.jsx)("h4",{children:"Evolutionary Data Analytics"}),(0,a.jsx)("p",{children:"The Kannan lab is driving significant advancements in evolutionary data mining and analytics in protein research. We have developed novel tools like the Multiple Sequence Alignment Ontology (MSAOnt) and KinView [1] for analyzing protein sequence alignments and visualizing complex data about proteins, such as their sequence variations, cancer-related variants, and post-translational modifications. Additionally, the team has discovered differential phosphorylation patterns in kinases, identifying novel regulatory features that could have implications for understanding disease. We developed Phosformer [2], a deep learning model that predicts kinase and substrate pairs based on sequence. We also developed KinOrtho [3], an advanced orthology inference method that maps kinase orthologs across various species. In another project, the lab launched GTXplorer [4], an online data analytics platform that connects sequence-structure-function relationships to the evolution of glycosyltransferases, facilitating comparative glycomics. These tools and platforms embody the lab's commitment to leveraging data mining and analytics to uncover more profound insights into protein function and structure."})]})}),(0,a.jsx)("div",{className:"4u 12u(narrower)",children:(0,a.jsx)("section",{id:"sidebar",className:"box",children:(0,a.jsxs)("section",{children:[(0,a.jsx)("h3",{children:"Selected Publications:"}),(0,a.jsxs)("ol",{children:[(0,a.jsxs)("li",{children:[(0,a.jsx)("a",{href:"https://doi.org/10.1039%2Fc6mb00466k",children:"KinView: a visual comparative sequence analysis tool for integrated kinome research"}),(0,a.jsx)("hr",{})]}),(0,a.jsxs)("li",{children:[(0,a.jsx)("a",{href:"https://academic.oup.com/bioinformatics/article/39/2/btad046/7000331",children:"Phosformer: an explainable transformer model for protein kinase-specific phosphorylation predictions"}),(0,a.jsx)("hr",{})]}),(0,a.jsxs)("li",{children:[(0,a.jsx)("a",{href:"https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04358-3",children:"KinOrtho: a method for mapping human kinase orthologs across the tree of life and illuminating understudied kinases"}),(0,a.jsx)("hr",{})]}),(0,a.jsxs)("li",{children:[(0,a.jsx)("a",{href:"https://doi.org/10.1093/glycob/cwab082",children:"GTXplorer: A portal to navigate and visualize the evolutionary information encoded in fold A glycosyltransferases"}),(0,a.jsx)("br",{}),(0,a.jsx)("br",{}),(0,a.jsx)("a",{href:"publications.html",className:"button alt",children:"View all publications"}),(0,a.jsx)("hr",{})]})]})]})})})]})]})})})}},function(e){e.O(0,[888,774,179],function(){return e(e.s=2682)}),_N_E=e.O()}]);